src/main/resources/data/malum/tags/blocks/cursedworkaroundjank.txt

```diff
@@ -0,0 +1 @@
+please remove this as soon as it can be done this is cursed voodoo
```

Comment:
Why have decay away in the tag in the first place?
grass/flowers behave the same way, when the grass_block below them is removed